client/src/components/imagine23/Main.js

```diff
@@ -4,7 +4,7 @@ import { navigate, Router } from "@reach/router";
 import { bindActionCreators } from "redux";
 import LandingPage from "./pages/landingPage";
 import MainInstructions from "./pages/mainInstructions";
-import Reading from "./pages/Reading";
+import { default as Reading } from "../body/Reading/Reading";
```

Comment:
you also could just have it export it as default from the file so you dont have to deal with this destructing in the import

Reply:
This has been handled.